.github/workflows/repository-dispatch.yml

```
@@ -0,0 +1,54 @@
+---
+# yamllint disable rule:truthy rule:line-length
+name: Trigger Infrahub update in other repositories
+
+on:
+  workflow_dispatch:
+    inputs:
+      runs-on:
+        description: "The OS to run the job on"
+        required: false
+        default: "ubuntu-22.04"
+        type: string
+      version:
+        type: string
+        required: false
+        description: The string to extract semver from.
+        default: ''
+  workflow_call:
+    inputs:
+      runs-on:
+        description: "The OS to run the job on"
+        required: false
+        default: "ubuntu-22.04"
+        type: string
+      version:
+        type: string
+        required: false
+        description: The string to extract semver from.
+        default: ''
+
+jobs:
+  repository-dispatch:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        # Etiher a literal path, or the name of a secret...
```

My small contribution to this great PR :P 
```suggestion
        # Either a literal path, or the name of a secret...
```